addons/project/tests/test_project_recurrence.py

```diff
@@ -19,7 +19,6 @@ def setUpClass(cls):
         cls.stage_b = cls.env['project.task.type'].create({'name': 'b'})
         cls.project_recurring = cls.env['project.project'].with_context({'mail_create_nolog': True}).create({
             'name': 'Recurring',
-            'allow_recurring_tasks': True,
```

Comment:
Can you check if we have a test to check the recurring_task field on all tasks are false when the `group_project_recurring_tasks` setting is disabled in the config?

If not, then can you add this test as well? Thanks in advance! 🙂